dio/test/utils.dart

```diff
@@ -14,6 +14,8 @@ Encoding requiredEncodingForCharset(String charset) =>
 /// The URL for the current server instance.
 Uri get serverUrl => Uri.parse('http://localhost:${_server?.port}');
 
+final isWeb = identical(1, 1.0);
```

Comment:
Using this might be more accurate. https://github.com/cfug/dio/blob/c2347f3bf74250d83307798ae77f0eb04febe38f/plugins/cookie_manager/lib/src/cookie_mgr.dart#L7-L9